rotors_gazebo_plugins/CMakeLists.txt

```diff
@@ -1,142 +1,422 @@
+# Optional arguments to be passed into file
+# ADDITIONAL_INCLUDE_DIRS           string  Additional include directories to add to every build target (PX4 uses this).
+# BUILD_MAVLINK_INTERFACE_PLUGIN    bool    Build mavlink_interface_plugin (requires mav dependency).
+# BUILD_OCTOMAP_PLUGIN              bool    Build the optical map plugin (requires Octomap).
+# BUILD_OPTICAL_FLOW_PLUGIN         bool    Build the optical flow plugin (requires OpenCV).
+# MAVLINK_HEADER_DIR                string  Location of MAVLink header files. If not provided, this CMakeLists.txt file will
+#                                               search the default locations (e.g. ROS) for them. This variable is only required
+#                                               if BUILD_MAVLINK_INTERFACE_PLUGIN=TRUE.
+# NO_ROS                            bool    Build without any ROS dependencies.
```

Comment:
Added defaults (for booleans, the strings already have their default cases handled).